index.html

```diff
@@ -1127,47 +1127,127 @@ <h2>Relative DID URLs</h2>
-<a href="#core-representations"></a>.
-        </p>
-    </section>
+data-cite="INFRA#map-entry">properties</a>, where each property consists of
+a property name/property value pair. The data model contains at least two
+different classes of properties. The first class of properties are called
+core properties, and are specified in section <a href="#core-properties"></a>.
+The second class of properties are called representation properties, and
+are specified in section <a href="#representations"></a>.
+    </p>
+    <p>
+All property names in the data model are <a
+data-cite="INFRA#strings">strings</a>. All property values are expressed
+using one of the data types in the table below.
+    </p>
+
+    <table class="simple" id="public-key-support">
+      <thead>
+        <tr>
+          <th>
+Data&nbsp;Type
+          </th>
+          <th>
+Considerations
+          </th>
+        </tr>
+      </thead>
+
+      <tbody>
+        <tr>
+          <td>
+<a data-cite="INFRA#maps">ordered&nbsp;map</a>
+          </td>
+          <td>
+A finite ordered sequence of key/value pairs, with no key appearing twice as
+specified in [[INFRA]].
+          </td>
+        </tr>
+        <tr>
+          <td>
+<a data-cite="INFRA#list">list</a>
+          </td>
+          <td>
+A finite ordered sequence of items as specified in [[INFRA]].
+          </td>
+        </tr>
+        <tr>
+          <td>
+<a data-cite="INFRA#ordered-set">ordered&nbsp;set</a>
+          </td>
+          <td>
+A <a data-cite="INFRA#list">list</a> that does not contain the same item twice
+as specified in [[INFRA]].
+          </td>
+        </tr>
+        <tr>
+          <td>
+<dfn>datetime</dfn>
+          </td>
+          <td>
+A date and time value that is capable of losslessly expressing all values
+expressible by a `dateTime` as specified in
+[<a data-cite="XMLSCHEMA11-2#dateTime">XMLSCHEMA11-2</a>].
+          </td>
+        </tr>
+        <tr>
+          <td>
+<a data-cite="INFRA#string">string</a>
+          </td>
+          <td>
+A sequence of code units often used to represent human readable language
+as specified in [[INFRA]].
+          </td>
+        </tr>
+        <tr>
+          <td>
+<dfn>decimal</dfn>
+          </td>
+          <td>
+A value that represents a subset of the real numbers, which can be
+represented by decimal numerals as specified in
+[<a data-cite="XMLSCHEMA11-2#decimal">XMLSCHEMA11-2</a>]. To maximize
+interoperability, implementers are urged to only express values capable of
+being represented by 53 bits in binary notation.
+          </td>
+        </tr>
+        <tr>
+          <td>
+<dfn>double</dfn>
+          </td>
+          <td>
+A value that is often used to approximate arbitrary real numbers as specified
+in [<a data-cite="XMLSCHEMA11-2#double">XMLSCHEMA11-2</a>]. To maximize
+interoperability, implementers are urged to only express values capable of
+being represented by 64 bits in binary notation.
```

Comment:
We might want to mention IEEE 754 here?

Reply:
XMLSCHEMA11-2 refers to IEEE754, I also pointed to the warning about numbers in https://tools.ietf.org/html/rfc7159#section-6.